### PR TITLE
Cook 2 user repo tests implementation

### PR DESCRIPTION
**Pull request by NemeCSK**
Please check my integration test, it was created before the milestone

### CODE REVIEW

File: Cookaholic_Backend/src/test/java/com/toolsofsoftwareprojects/cookaholic_backend/service/UserServiceTests.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    private MockMvc mockMvc;
+
+    // given/when/then format - BDD style
+    @Test
+    public void givenUsers_whenGetAllUser_thenListOfUsers() throws Exception {
+        // given - setup or precondition
+        List<CookaholicUser> users =
+                List.of(CookaholicUser.builder()
+                            .email("daniel@a.com")
+                            .userTitle("JavaScript")
```
Comment (kapchx) [inline Cookaholic_Backend/src/test/java/com/toolsofsoftwareprojects/cookaholic_backend/service/UserServiceTests.java]: user title doesn't exist anymore

**Comment by kapchx** [comment]
Delete the added user after integration test

**Comment by NemeCSK** [comment]
> Delete the added user after integration test

Why? im just running the one integration test by itself

File: Cookaholic_Backend/src/test/java/com/toolsofsoftwareprojects/cookaholic_backend/service/UserServiceTests.java (hunk 1 above)
Comment (NemeCSK) [reply Cookaholic_Backend/src/test/java/com/toolsofsoftwareprojects/cookaholic_backend/service/UserServiceTests.java]: changed thanks

**Comment by kapchx** [comment]
Because you have to delete the user from postgresql database

**Comment by NemeCSK** [comment]
> Because you have to delete the user from postgresql database

True! added jira ticket for extension

**Pull request merged**